torch-test-model.ipynb

```diff
@@ -67,14 +125,21 @@
     "        return output"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "ced04b88",
+   "metadata": {},
+   "source": [
+    "Then, load the model onto the GPU along with the trained weights from the `torch-use-gpu.ipynb` workbook."
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "id": "e4c24aa3-0b70-4290-8e70-c1ef30775f76",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# putting the model on our GPU and loading in those trained weights.\n",
     "model = SimpleNet().to( device )\n",
     "model.load_state_dict( torch.load(\"mnist_fashion_SimpleNet.pt\") )"
    ]
```

Comment:
@cam-garrison After this point is where there's info missing from the gdoc

Reply:
Cam mentioned on the GDoc that we can ignore this, as there are enough code comments to get users through the process.